packages/common/src/hooks/purchaseContent/usePurchaseContentFormConfiguration.ts

```diff
@@ -69,26 +66,18 @@ export const usePurchaseContentFormConfiguration = ({
         customAmount
       })
 
-      if (purchaseMethod === PurchaseMethod.CARD) {
-        dispatch(
-          startPurchaseContentFlow({
-            extraAmount,
-            extraAmountPreset: amountPreset,
-            contentId: track.track_id,
-            contentType: ContentType.TRACK
-          })
-        )
-      } else if (purchaseMethod === PurchaseMethod.EXISTING_BALANCE) {
+      if (purchaseMethod === PurchaseMethod.MANUAL_TRANSFER) {
```

Comment:
I flipped this conditional around since "existing balance" and "card" will follow the same flow, with manual funding requiring an additional step in the component before dispatching the purchaseContent action.